.pre-commit-config.yaml

```diff
@@ -1,7 +1,10 @@
+# https://github.com/psf/black/blob/master/.pre-commit-config.yaml
```

Comment:
Just clicked into the url and saw this line:
```
# Note: don't use this config for your own repositories. Instead, see
# "Version control integration" in README.md.
```
Can you make sure it is okay to follow its config?

Reply:
interesting. I missed it..

Reply:
Nice catch! Thanks! I did a bit research, although they don't suggest to use it, it does suit our use case (somehow):

in pre-commit doc: https://pre-commit.com/#repository-local-hooks

The doc states when we should use `local` repo instead of what black's "Version control integration" suggests:

> Hooks require state that is only present in a built artifact of your repository (such as your app's virtualenv for pylint).

And the example they gave (for `pylint`) is quite similar to what we are using right now, except for `require_serial` (if true this hook will execute using a single process instead of in parallel), which does not make too much a difference (I just removed it)

```
    -   id: pylint
        name: pylint
        entry: pylint
        language: system
        types: [python]
```

Reply:
looks good to me :)